examples/CMakeLists.txt

```diff
@@ -16,35 +16,41 @@ target_link_libraries(big_matrix PRIVATE pc2l_lib)
 target_link_libraries(big_matrix ${MPI_CXX_LIBRARY})
 
 
-add_executable(std_matrix std_matrix.cpp STDMatrix.cpp)
-
-target_compile_features(std_matrix PRIVATE cxx_std_11)
-
-target_link_libraries(std_matrix PRIVATE pc2l_lib)
-
-target_link_libraries(std_matrix ${MPI_CXX_LIBRARY})
-
+# add_executable(std_matrix std_matrix.cpp STDMatrix.cpp)
+#
+# target_compile_features(std_matrix PRIVATE cxx_std_11)
+#
+# target_link_libraries(std_matrix PRIVATE pc2l_lib)
+#
+# target_link_libraries(std_matrix ${MPI_CXX_LIBRARY})
+#
  #Web Cache example
-
-add_executable(web_cache web_cache.cpp Matrix.cpp ChildProcess.cpp HTTPFile.cpp)
-
-target_compile_features(web_cache PRIVATE cxx_std_11)
-
```

Comment:
not building the other examples right now. Some of the Boost ASIO stuff was causing build issues